q2_feature_classifier/_consensus_assignment.py

```diff
@@ -0,0 +1,229 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import tempfile
+import subprocess
+import pandas as pd
+from os.path import isfile
+from collections import Counter
+
+
+def _consensus_assignments(cmd, ref_taxa, t_delim=';', min_consensus=0.51,
+                           unassignable_label="Unassigned"):
+    '''Run command on CL and find consensus taxonomy.'''
```

Comment:
Can you expand `CL` rather than abbreviate it (I'm not sure what you're referring to here).